.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: user-bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Windows]
+ - Version [e.g. 10]
```

Comment:
Suggestion:

**Environment (fill in the correct information of the system on which you observed the bug):**

- **Operating system:**
Windows 10 / Ubuntu 18.04 / macOS 10.15 / ...

- **IntelliJ IDEA version:**
2020.2 Community

- **Version of A+ Courses plugin:**
1.0.0